src/Syncromatics.Clients.Metro.Api/MetroApiClient.cs

```diff
@@ -60,5 +91,22 @@ public Task<List<Stop>> GetStopsByRouteId(string routeId)
         {
             return _client.GetStopsByRoute(routeId);
         }
+
+        private static string ToEnumValue(Enum stopPassageOption)
+        {
+            var memberName = stopPassageOption.ToString();
+            var declaredMembers = stopPassageOption.GetType()
+                .GetTypeInfo()
+                .DeclaredMembers;
+            var customAttributes = declaredMembers
+                .Where(m => m.Name == memberName)
+                .SelectMany(m => m.CustomAttributes);
+            var enumMemberAttributes = customAttributes
+                .Where(a => a.AttributeType == typeof(EnumMemberAttribute))
+                .SelectMany(a => a.NamedArguments)
+                .Where(na => na.MemberName == "Value")
+                .Select(na => na.TypedValue.Value as string);
+            return enumMemberAttributes.SingleOrDefault() ?? memberName;
```

Comment:
memberName is potentially a valid value for these?

Reply:
`memberName` is the enum value (i.e., `AnyBoarding`) as a string. This would be a problem if the enum were a bitmask/flag, but it's not in this case.